Src/Couchbase.Linq/Execution/BucketQueryExecutor.cs

```diff
@@ -257,9 +259,9 @@ public T ExecuteSingle<T>(QueryModel queryModel, bool returnDefaultWhenEmpty)
             return result;
         }
 
-        public async Task<T> ExecuteSingleAsync<T>(LinqQueryRequest queryRequest)
+        public async Task<T> ExecuteSingleAsync<T>(LinqQueryRequest queryRequest, CancellationToken cancellationToken)
         {
-            var resultSet = await ExecuteCollectionAsync<T>(queryRequest);
+            var resultSet = await ExecuteCollectionAsync<T>(queryRequest, cancellationToken);
```

Comment:
I think this should this have a ConfigureAwait(false) too.

Reply:
Done